portfolio/src/main/webapp/sheetsApi.js

```diff
@@ -0,0 +1,146 @@
+// Client ID and API key from the Developer Console
+var CLIENT_ID =
+    '499747085593-hvi6n4kdrbbfvcuo1c9a9tu9oaf62cr2.apps.googleusercontent.com';
+var API_KEY = 'AIzaSyAcS8-F743Xy2ZTSYxxZIB0nY5kDQQ8i7E';
```

Comment:
I don't think our API Key should be public.

Reply:
Done.